azure-pipelines.yml

```
@@ -207,7 +207,8 @@ stages:
           include:
           - src/Compilers/*
           - src/Dependencies/*
-          - eng/*
+          - src/NuGet/Microsoft.Net.Compilers.Toolset/*
+          - eng/*`
```

Not intended I guess?

```suggestion
          - eng/*
```

---

whoops